src/interfaces/suggestion.ts

```diff
@@ -1,11 +1,13 @@
-interface IRole {
-  filledBy: string
+export interface IRole {
+  filledBy: string[] | null
   instrument: string
+  note: string | null
 }
 
 export interface ISuggestion {
   id: string
   artists: string[]
+  date: string
```

Comment:
Why a string? I believe Firebase has a timestamp type right? Will probably be easier to have with sorting